composer.json

```diff
@@ -11,7 +11,7 @@
     "require": {
         "php": ">=5.5.0",
         "league/flysystem": "~1.0",
-        "microsoft/azure-storage": "~0.10.1"
+        "microsoft/azure-storage": "~0.15.0"
```

Comment:
Is this required for the emulator?

Reply:
not for emulator, this is for implement `getUri()`